### PR TITLE
Add recipe for python-logstash-async

### PR DESCRIPTION
**Pull request by tiefling-cat**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/python-logstash-async/meta.yaml
Comment (dopplershift) [inline recipes/python-logstash-async/meta.yaml:23]: Upstream lists `limits`. Any reason not to add it here?

File: recipes/python-logstash-async/meta.yaml
Comment (dopplershift) [inline recipes/python-logstash-async/meta.yaml:28]: Since upstream `__init__.py` doesn't import much, it would be good to add something like 
```suggestion
    - logstash_async
    - logstash_async.worker
```
which would have found the missing `limits`.

**Review by dopplershift — CHANGES_REQUESTED** [review]

File: recipes/python-logstash-async/meta.yaml
Comment (tiefling-cat) [reply recipes/python-logstash-async/meta.yaml:23]: > Upstream lists `limits`. Any reason not to add it here?

@dopplershift the problem with `limits` is that it's already hosted in another conda channel (see [https://anaconda.org/equipoise/limits](https://anaconda.org/equipoise/limits)), but I didn't find any option to specify additional channels besides `conda-forge` (which, as I understand, may be by design).

Is there a known way to do that? Or should I just push `limits` into `conda-forge` as well ignoring the fact that it has its own channel? Please advise.

Comment (dopplershift) [reply recipes/python-logstash-async/meta.yaml:23]: You'll need to add `limits` to conda-forge, either in this PR (might need to redo some builds after merging due to issues of which gets built first) or a separate PR. The conda-forge packages do not and cannot depend on packages in any other channel.

Comment (tiefling-cat) [reply recipes/python-logstash-async/meta.yaml:23]: Please review the pull request for limits ([https://github.com/conda-forge/staged-recipes/pull/17489](https://github.com/conda-forge/staged-recipes/pull/17489) at your convenience.

**Comment by tiefling-cat** [comment]
@dopplershift since limits are in conda-forge now, please review at your convenience and approve if everything else is fine.

File: recipes/python-logstash-async/meta.yaml
Comment (dopplershift) [reply recipes/python-logstash-async/meta.yaml:29]: Ping here.

Comment (tiefling-cat) [reply recipes/python-logstash-async/meta.yaml:29]: > Ping here.

Ughm, ping? @dopplershift I believe now it must be ready to merge.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**
